src/pages/studyView/StudyViewPageStore.ts

```diff
@@ -2224,6 +2239,49 @@ export class StudyViewPageStore
         }
     }
 
+    parseRawFilterJson(filterJson: string): any {
+        let rawJson;
+        try {
+            rawJson = JSON.parse(decodeURIComponent(filterJson));
+        } catch (e) {
+            console.error('FilterJson invalid Json: error: ', e);
+        }
+        return rawJson;
+    }
+
+    async fetchSamplesWithStudyIds(studyIds: string[]) {
+        let studyViewFilter: StudyViewFilter = {} as any;
+        studyViewFilter.studyIds = studyIds;
+        return await internalClient.fetchFilteredSamplesUsingPOST({
+            studyViewFilter: studyViewFilter,
+        });
+    }
+
+    convertPatientIdentifiersToSampleIdentifiers(
+        patientIdentifierFilter: PatientIdentifierFilter,
```

Comment:
this should accept patientIdentiviers, not the filter since i think all that's important is the identifiers, not the parent of them.  clearer interface.

Reply:
done